earthaccess/exceptions.py

```diff
@@ -0,0 +1,13 @@
+class LoginStrategyUnavailable(Exception):
+    """The selected login strategy was skipped.
+
+    This should be raised when a login strategy can't be attempted, for example because
+    "environment" was selected but the envvars are not populated.
+
+    DO NOT raise this exception when a login strategy is attempted and failed, for
+    example because credentials were rejected.
+    """
```

Comment:
```suggestion
    """
    pass
```

Reply:
This isn't necessary since the docstring counts as the function body. If you still feel strongly about including `pass`, let me know!

Reply:
Right, I realize it's not necessary, but it may be confusing to others, so I suggest adding `pass` to avoid potential head-scratching by others who might stumble upon the code and wonder about its validity.

Reply:
No worries! Adding.